### DISCUSSION TITLE
Sequence number for messages

### DISCUSSION THREAD

**Discussion by benaish** [TrakHound/MTConnect.NET#54] [Q&A]
Is it possible to configure the SHDR-MQTT-Agent to send out an incremental sequence number in the messages?

**Comment by PatrickRitchie** [comment]
The MQTT functionality is being changed in the Version 6 release. This will conform better to the HTTP protocol and will be compatible with the c++ Agent.

You can try out the new agent in the Version 6 branch : https://github.com/TrakHound/MTConnect.NET/releases/tag/untagged-1926fb8a6fc4ed4c6b60
